proposals/spline-animation/README.md

```diff
@@ -0,0 +1,1980 @@
+last knot in a series.  Evaluation in the extrapolated region of such a series
+will return no value.  It will also be possible to disable _only_ extrapolation
+by using the "None" extrapolation mode.
+
+Series will introduce an additional pattern that can block weaker opinions: an
+empty series.  As always, the presence of any series opinions overrides weaker
+layers and default values.  A series with no knots is still a valid series, so
+it blocks weaker opinions, but it provides no values.  Like `SdfValueBlock`, an
+empty series will not block fallback values when `UsdAttribute::Get` is called.
+
+## Serialization
+
+Series will become part of the Sdf data model, and thus supported in the `usda`
+and `usdc` file formats.
+
+## Scalar Transforms
+
+As an auxiliary improvement, we will add scalar translation and scaling to
+`UsdGeomXformOp`.  This will bring translation and scaling up to the level of
+rotations, which can already be expressed as scalars.  This improvement will
+mean that translation, for example, can be expressed as three different ops - an
+X translation, a Y translation, and a Z translation - rather than only as a
+single vector-valued XYZ translation.
+
+Transforms are one of the most common cases for spline-driven animation, so we
+want to make it convenient to drive transforms from scalar splines.
+
+## Series Utilities
+
+There are a few utilities for series introspection and manipulation that are
+sufficiently low-level and general to merit inclusion in USD Anim.
+
+For any series:
+
+- A **diff** utility that finds time regions that differ between two series.
+
+For splines only:
+
+- A **simplify** utility that eliminates knots that have little effect.
+- A **resample** utility that generates a simpler set of knots.
+
+The current implementations of the simplify and resample utilities always return
+all-Bezier splines.  It is TBD whether this can be improved; for example, it
+seems desirable that an all-Hermite input spline should result in an all-Hermite
+output spline.
+
+## Series in usdview
+
+The `usdview` application will be updated to display series values, including a
+basic visualization of curves over time.
+
+## Motion Blur
+
+When rendering animated USD content with motion blur, we must determine the time
+coordinates at which samples should be taken.  This can include both how many
+different samples should be taken for a given frame, and how the samples are
+distributed in time.
+
+In existing USD content, animated values are represented with time samples.
+Time samples provide clear policy about render sampling: we sample at the time
+sample times.
+
+When rendering USD content containing splines, the situation will be different.
+We could potentially use knot times for sampling, but knot times don't
+necessarily indicate an intent to sample; they may only have been chosen in
+order to achieve a desired curve shape with sparse data.
```

Comment:
Without an API to do this, I think we'd be forced to bake out the full curve or segments at the resolution we needed to capture motion blur samples.

Reply:
Yes.  We're still cogitating on this detail, but for now my best guess is that, for Presto at least, we'll bake out time samples for these cases.